### PR TITLE
port control_performance_analysis

### PR DESCRIPTION
**Pull request by k0suke-murakami**
Signed-off-by: kosuke murakami <kosuke.murakami@tier4.jp>

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Read [pull request guidelines][pull-request-guidelines]
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: control/control_performance_analysis/launch/controller_performance_analysis.launch.xml [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
 
-  <arg name="control_performance_analysis_param_path"
-       default="$(find-pkg-share control_performance_analysis)/config/control_performance_analysis.param.yaml" />
+  <arg name="control_performance_analysis_param_path" default="$(find-pkg-share control_performance_analysis)/config/control_performance_analysis.param.yaml" />
   <arg name="input/reference_trajectory" default="/planning/scenario_planning/trajectory" />
   <arg name="input/control_raw" default="/control/trajectory_follower/lateral/control_cmd" />
-  <arg name="input/measured_steering" default="/vehicle/status/steering" />
-  <arg name="input/current_velocity" default="/localization/twist" />
+  <arg name="input/measured_steering" default="/vehicle/status/steering_status" />
+  <arg name="input/current_odometry" default="/localization/odometry" />
```
Comment (TakaHoribe) [inline control/control_performance_analysis/launch/controller_performance_analysis.launch.xml:7]:
```suggestion
  <arg name="input/current_odometry" default="/localization/kinematic_state" />
```

File: control/control_performance_analysis/package.xml [hunk 2]
```diff
@@ -10,12 +10,14 @@
   <buildtool_depend>ament_cmake_auto</buildtool_depend>
   <buildtool_depend>rosidl_default_generators</buildtool_depend>
 
-  <depend>autoware_control_msgs</depend>
+  <depend>autoware_auto_control_msgs</depend>
+  <depend>autoware_auto_planning_msgs</depend>
+  <depend>autoware_auto_vehicle_msgs</depend>
   <depend>autoware_planning_msgs</depend>
```
Comment (TakaHoribe) [inline control/control_performance_analysis/package.xml:16]: can we remove '<depend>autoware_planning_msgs</depend>' ?

File: control/control_performance_analysis/launch/controller_performance_analysis.launch.xml (hunk 1 above)
Comment (k0suke-murakami) [reply control/control_performance_analysis/launch/controller_performance_analysis.launch.xml]: Thanks, I changed the topic name. https://github.com/tier4/AutowareArchitectureProposal.iv/pull/698/commits/67e6c9107939d403aa16f9655a0a0a3feaecafa9

File: control/control_performance_analysis/package.xml (hunk 2 above)
Comment (k0suke-murakami) [reply control/control_performance_analysis/package.xml]: Thanks, remove the unnecessary dependency https://github.com/tier4/AutowareArchitectureProposal.iv/pull/698/commits/bad25fd1c825e292b812f56425005a4af05171ce

**Review by TakaHoribe — APPROVED** [review]

**Review by boyali — APPROVED** [review]
Murakami-san, thank you very much for porting. I haven't seen any problem too.

**Pull request merged**